### PR TITLE
Fixes #589 - Scottish iTEC Profile

### LINKED ISSUE #589: Add iTEC for Scottish

### PR DESCRIPTION
**Pull request by SamLefevre**
Fixes #589 

# Summary of changes

Add iTEC profile, tweaked symbology from VATSIM Germany (EDUU).

# Screenshots (if necessary)

image

### CODE REVIEW

File: UK/Data/ASR/PX_AC/AC1T.asr [hunk 1]
```diff
@@ -1,57 +1,8 @@
...
-ARTCC boundary:LFP28:
-ARTCC boundary:LFP32:
-ARTCC boundary:LFP33:
-ARTCC boundary:LFP6-1:
-ARTCC boundary:LFP6-2:
-ARTCC boundary:LFP7:
-ARTCC boundary:LFP81:
-ARTCC boundary:LFR10:
+SECTORFILE:UK\Data\Sector\UK_2023_10.sct
+SECTORTITLE:UK_2023_10.sct
```
Comment (luke11brown) [inline UK/Data/ASR/PX_AC/AC1T.asr:5]: Remove

File: UK/Data/ASR/PX_AC/AC2T.asr [hunk 2]
```diff
@@ -1,57 +1,8 @@
...
-ARTCC boundary:LFP28:
-ARTCC boundary:LFP32:
-ARTCC boundary:LFP33:
-ARTCC boundary:LFP6-1:
-ARTCC boundary:LFP6-2:
-ARTCC boundary:LFP7:
-ARTCC boundary:LFP81:
-ARTCC boundary:LFR10:
+SECTORFILE:UK\Data\Sector\UK_2023_10.sct
+SECTORTITLE:UK_2023_10.sct
```
Comment (luke11brown) [inline UK/Data/ASR/PX_AC/AC2T.asr:5]: Remove

**Review by luke11brown — COMMENTED** [review]
Maybe worth asking Juha which files can be pointed at other locations to reduce the amount of duplicate files.

When testing myself, some were not able to have locations.

**Comment by SamLefevre** [comment]
Symbology updated for readability and tags updated with updated trend arrows and TopSky releases.

**Comment by SamLefevre** [comment]
image

**Comment by luke11brown** [comment]
I'd like to see a retained NERC profile for Scottish, as you've already said, it looks difficult to control top down with iTEC.

**Comment by SamLefevre** [comment]
Hmm not too sure here, I agree we can give people the option but I don't think it's harder to control top-down when I done it anyways, the only thing that you have to hover over the tag for is cleared speed, heading/nav is on tagged and so is cleared level.

**Comment by luke11brown** [comment]
I think with all the duplicated files for TopSky, we're going to need a method of keeping parity between those used by iTEC and NERC files. I suggest that we look at some sort of automation, or editing an existing one to ensure that on every release both files (e.g. TopSkyAres.txt) have exactly the same content.

**Comment by SamLefevre** [comment]
Recent TopSky changes have been reflected here.

**Review by luke11brown — APPROVED** [review]

**Pull request merged**